tests/scattering1d/test_torch_backend_1d.py

```diff
@@ -252,3 +252,28 @@ def coefficent(n):
     print(z_2.shape)
     assert not z_2.shape[-1] == 2
     assert torch.allclose(x_r, z_2)
+
+
+@pytest.mark.parametrize("device", devices)
+@pytest.mark.parametrize("backend", backends)
+def test_transpose_1d(device, backend, random_state=42):
+    """
+    Tests the correctness and differentiability of pad_1d
+    """
+    shape = (10, 20, 3, 5, 2)
+    shape_T = (10, 20, 5, 3, 2)
+
+    x = torch.ones(shape)
```

Comment:
i would prefer x to be [1, 2, 3, 4, 5, 6 ....] reshaped to `shape`
instead of [1, 1, 1, 1, 1, 1, ... ]

Reply:
fixed